src/StaticFAQ/FAQCategoryList.js

```diff
@@ -72,12 +73,13 @@ const querySubcategory = graphql`
 class FAQCategoryList extends React.Component<Props> {
   renderFAQArticlePerexes = (
     faqs: $ReadOnlyArray<?FAQArticlePerexFragment>,
+    category: FAQCategory_category,
   ) => {
     return (
       <div>
         {faqs
           .filter(Boolean)
-          .map(faq => <FAQArticle key={faq.id} article={faq} />)}
+          .map(faq => <FAQArticle key={faq.id} article={faq} category={category} />)}
```

Comment:
if you add category to `FAQArticle`, you have to add it also to the `FAQArticle` on `SearchAllFAQs.js`.